myWork/lib/readFile.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+const fs = require('fs');
+// module.exports = function(err, data) {
```

Comment:
Just a quick note. It's bad form to include a bunch of commented out code in your commits.